packages/dashboard/src/app/views/myStories/content/index.js

```diff
@@ -18,7 +18,7 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { __, sprintf } from '@googleforcreators/i18n';
+import { sprintf, __ } from '@googleforcreators/i18n';
```

Comment:
?

Reply:
As part of the feedback given, I had to use `sprintf` again, and I just put it in a different order when I imported it to address feedback vs. the original code... that's all.

Reply:
So just revert that change, leave in it's original order.

Reply:
IDK, the order doesn't matter and the change is already done. No big deal.